boot-microservices/src/main/java/org/ow2/proactive/boot/microservices/IAMStarter.java

```diff
@@ -107,6 +116,17 @@ public static Process start(String paHome, String iamMicroservicesPath, String i
 
             LOGGER.debug(streamOutput(process.getInputStream()));
 
+            /*
```

Comment:
The format of this comment looks a bit weird.

Reply:
I agree. I just wanted to mark the post-startup operations with a specific comment. I'll try to find a better solution.

Reply:
After investigating. This seems the right way to add a multi-line comment inside a method. We use '/*' with only one star ('/**' is a documentation comment that can be added only outside methods).